framework/src/abi/schema.ts

```diff
@@ -373,6 +373,10 @@ export const verifyTransactionRequestSchema = {
 			fieldNumber: 2,
 			...transactionSchema,
 		},
+		header: {
```

Comment:
with this the request will accept a full header object which we don't want to pass